index.html

```diff
@@ -13,6 +14,8 @@
             <h1 class="banner-title">Dasmoto's Arts & Crafts</h1>
         </div>
 
+        <!-- Brushes Section -->
```

Comment:
Good practice to comment sections. If you saw the solution from codecademy they commented every section.